### ISSUE TITLE
UI: compare view: remove benchmark results that do not show 'contender value'?

### ISSUE THREAD

**Issue by jgehrcke** [conbench/conbench#1064]
Scenario: comparison of two runs.

Example:
![image](https://user-images.githubusercontent.com/265630/230314030-932be30f-d85e-434f-b4c7-8d9272697b53.png)

Action items:
- First, understand why for the first three rows there is no value in the Contender column. Is it because these are for benchmark results in the baseline run that simply have no meaningful comparison in the contender run? 
- If there is a good reason to not have these values: either do not display these rows, or inform the user about the reason for the missing value

**Comment by jonkeane** [comment]
Could you post a link to where you got the screen shot  — I would love to dig in there?

**Comment by jgehrcke** [comment]
> Could you post a link to where you got the screen shot — I would love to dig in there?

No, this was in my local dev environment using database state created with `make db-populate`, i.e with https://github.com/conbench/conbench/blob/43a11e200a7c2753788b5aba391c59e035a3a4c0/conbench/tests/populate_local_conbench.py. 

This is likely to exercise special cases that we do not often see otherwise. But these special cases might give valuable inspiration for how-to-build-things.
